File: Listas/lista_fabio/02/condicionais/q19_calcular-imc.js
import {print, input} from '../../../../io_utils.js'

function main() {
    print("------- Cálculo de IMC -------\n")


    // entrada
    const altura = Number(input("Altura (metros): "))
    const peso = Number(input("Peso (kg): "))

    // processamento
    const imc = Math.trunc(peso / altura*altura)

    // saída
    if (imc < 25) {
        print(`\nIMC: ${imc}\nPeso normal.`)
    } else if (imc >= 25 && imc <= 30) {
        print(`\nIMC: ${imc}\nObeso.`)
    } else {
        print(`\nIMC: ${imc}\nObesidade mórbida.`)
    }
}

main()

File: Listas/lista_fabio/02/condicionais/q19_calcular-imc.py
def main():
    print("------- Cálculo de IMC -------\n")

    altura = float(input("Altura (metros): "))
    peso = float(input("Peso (kg): "))

    imc = peso // (altura*altura)

    if (imc < 25):
        print(f'\nIMC: {imc}\nPeso normal.')
    elif (imc >= 25 and imc <= 30):
        print(f'\nIMC: {imc}\nObeso.')
    else:
        print(f'\nIMC: {imc}\nObesidade mórbida.')

main()
